frontend/javascripts/oxalis/model/bucket_data_handling/wkstore_adapter.js

```diff
@@ -171,22 +171,19 @@ export async function requestFromStore(
     const errorMessage = `Requesting data from layer "${
       layerInfo.name
```

Comment:
One thing I noticed: Having the layer name in the error message inhibits the aggregation in airbrake. Maybe we can omit the layer name when sending the message to airbrake?

Reply:
For reference, airbrake is able to figure out that the layer name is dynamic and abstracts from that, so the aggregation still works :+1: